src/schemas/documentSchema.ts

```diff
@@ -0,0 +1,19 @@
+import * as yup from 'yup'
+
+export const documentSchema = yup
+  .object({
+    family_import_id: yup.string().required(),
+    variant_name: yup.string().optional(),
+    role: yup.string().required(),
+    type: yup.string().required(),
+    title: yup.string().required(),
+    source_url: yup.string().required(),
+    user_language_name: yup.string().required(),
+    // user_language_name: yup
```

Comment:
Is this eventually going to supersede line 11?

Reply:
Yes if we go with the react-chakra-select